### PR TITLE
Fix Kenya Regions

### PR DESCRIPTION
**Pull request by KenMunene**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: Corrected Kenyan regions to reflect the correct and more appropriate counties

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Edit a contact, and in State, the options for Kenya are very limited.  See https://en.wikipedia.org/wiki/ISO_3166-2:KE for official list of Counties.

#### Steps to test this PR:
1. Confirm limited set of five options as above
2. Apply this PR
3. Clear cache
4. Edit contact, and you should see the full list of counties

### CODE REVIEW

File: app/bundles/CoreBundle/Assets/json/regions.json [hunk 1]
```diff
@@ -2178,11 +2178,53 @@
...
+		"Lamu County",
+		"Machakos County",
+		"Makueni County",
+		"Mandera County",
+		"Marsabit County",
+		"Meru County",
+		"Migori County",
+		"Mombasa County",
+		"Murang'a County",
+		"Nairobi County",
```
Comment (RCheesley) [inline app/bundles/CoreBundle/Assets/json/regions.json]: Should this perhaps be Nairobi City, per the list here: https://en.wikipedia.org/wiki/ISO_3166-2:KE?

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for this PR!

For anyone testing it’s super easy.

Open a contact, select the state and when you type Kenya you’ll see only five options. 

screenshot-local mautictest-2019 11 14-18_46_44

This PR adds all the options that should be available.  After applying and clearing the cache you should see a much more extensive list.

screenshot-local mautictest-2019 11 14-18_48_38

I only had one query @KenMunene which is that in the official ISO list, Nairobi is listed as 'Nairobi City' so I wondered if this should be 'Nairobi City' in the list, rather than 'Nairobi County'.

Any thoughts? (I added an in-line comment on the code).

**Comment by RCheesley** [comment]
I made a small edit to the original post to include test instructions @KenMunene :)

File: app/bundles/CoreBundle/Assets/json/regions.json (hunk 1 above)
Comment (KenMunene) [reply app/bundles/CoreBundle/Assets/json/regions.json]: Seem to be used interchangeably. Happy to go with 'Nairobi City County' given official site uses this -> https://nairobi.go.ke/

**Review by RCheesley — APPROVED** [review]

**Comment by RCheesley** [comment]
@KenMunene thanks for making the change and apologies for taking so long to get around to approving!

**Review by dennisameling — APPROVED** [review]

**Pull request merged**